store/postgres_migration_test.go

```diff
@@ -1,151 +1,147 @@
 package store
 
 import (
-	"database/sql"
+	"context"
 	"fmt"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
-	"github.com/valocode/bubbly/config"
 	"github.com/valocode/bubbly/env"
-
-	"github.com/ory/dockertest/v3"
-	"github.com/ory/dockertest/v3/docker"
-	"github.com/valocode/bubbly/api/core"
-
-	"github.com/stretchr/testify/require"
+	"github.com/valocode/bubbly/test"
 )
 
 const (
 	postgresDatabase = "bubbly"
 	postgresUser     = "postgres"
 	postgresPassword = "secret"
-
-	cockroachDatabase = "defaultdb"
-	cockroachUser     = "root"
-	cockroachPassword = "admin"
```

Comment:
❓ Is this... the time we pull the plug on (now dysfunctional) CockroachDB support?
✔️ I would delete `store/cockroachdb.go`

Reply:
I think we can do that yes. After this PR. The tests in here were being skipped already anyway...